### DISCUSSION TITLE
Request of information

### DISCUSSION THREAD

**Discussion by InIure** [rustdesk/rustdesk#5110] [Q&A]
Hello,

My name is Kirill, i am a lawyer from Russian Federation.

I am really sorry for possible incorrect use of English.

This is the topic requiring in-depth analysis, so i wish some of developers will see it, or someone with the same level of knowledge, who will study the topic and give a constructive answer.

And i wish to say - there are no any complaints/grievance to/on developers of Rust Desk.

The essence of the matter: not so long ago, cases of fraud using the Rust Desk app became more frequent in Russia.

One man was approached me because he was deceived by scammers. Scammers used Rust Desk to get phone access, took the loan, and transferred the money to their accounts.

The fraudulent scheme looked realistic, so i sorry to him, and do not blame him for that.

Just 5 minutes - he got money to bank account, which was immediately transferred.

And now he need to pay 12 000$ to bank next five years, which is a very significant amount by the standards of a remote region of Russia (Aktyubinsky urban-type settlement, Tatarstan).

He have wife and two little childrens (5 and 3 years old), pays on a mortgage loan. It's incredibly hard to deal with this situation for him.

I agreed to provide him with free legal assistance.

In the Romano-German (and Russian) legal system, the law proceeds from the fact that you don't have to know about all the ways of fraud. On the other hand, bank (as a professional market participant) should have been more conscientious when issuing a loan and transferring funds.

In order to get rid of the loan (and force the bank to look for fraudster on its own), you have to prove that it was issued by fraudster. But in Rust Desk case, it all looks like he took the loan by his own.

So the question: is there any opportunity to show the court any evidence showing the use of Rust Desk for loan processing in the bank's mobile application?

If yes - i need detailed instructions for dummies, because this deceived man does not have the money to pay for the services of an IT specialist, so I will carry out these actions independently.

If no - thank you very much anyway for studying the issue and trying to find a solution.

Model of phone: Xiaomi redmi 10.
Version of Rust Desk app: current on June 7, 2023.
Version of bank app: "VTB Online" (app of "VTB Bank", Russia), current on June 7, 2023.

If more information is needed, ask me please.

**Comment by dinger1986** [comment]
Hello, this is indeed sad however it would be hard to prove 100% other than proving that RustDesk was installed on a certain date at a certain time which hopefully would correspond to the time the loan was taken out or should at least! 

https://android.stackexchange.com/questions/25263/where-can-i-find-out-when-i-installed-an-app has some guides.

Beyond that I don't really know what else to suggest other than, there will be logs on which might show connections, the guide above should show you how to access the root file system and from there you can hopefully find logs

**Comment by SergeyMy** [comment]
1. нужны настройки из приложения RustDesk 
. 
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/e5e0b388-9021-45c8-9a42-604b0dc321e1)
если сервера публичные - у владельца сервера должны остатся логи активности приложения,в том числе и адреса кто к кому и когда подключался.

PS с высокой долей вероятности так оно и есть, сомневаюсь что мошенники еще и диктовали жертве адреса своих серверов. так что если там все по дефолту то - спрошайте у разработчиков данного ПО (владельцев "rs-ny.rustdesk.com", "rs-sg.rustdesk.com")
PSPS как минимум нужен ID телефона (тот что сообщили мошенникам) - мо нему можно будет проверить логи серверов.
